widget/embedded/src/store/quote.ts

```diff
@@ -38,7 +37,13 @@ type SomeQuoteState = {
   error: QuoteError | null;
   warning: QuoteWarning | null;
 };
-
+export type SwapRequest = {
```

Comment:
It's not swap, its' all about quote. you can rename it to `RetryQuote`

Reply:
this comment can be applied to all other places you used `swap` or `swapRequest`. please update them accordingly.

Reply:
fixed